### ISSUE TITLE
inventory and other GUIs (I tried chest, golden bag, bagginses bag, mekanism, ender i/o) are partially transparent

### ISSUE THREAD

**Issue by croftyraider** [BluSunrize/ImmersiveEngineering#1366]
And because of this nothing looks right. Mekanism energy cube becomes unreadable. Etc.
I pointed the finger at your mod because I had just added it, and then I removed it and relaunched and the problem was gone.

**Comment by BluSunrize** [comment]
Please read the contribution guidelines and provide screenshots and modlist. Pretty sure that this is not on me.

**Comment by croftyraider** [comment]
Well, as I said, I tested by just adding/removing your mod and the problem appeared then disappeared (upon removal). Here is the mod list:

AppleCore-mc1.10.2-1.3.5.jar*
autopackager-1.6.3.jar*
Bagginses-1.10-3.0.2a.jar*
Baubles-1.9.4-1.2.1.0.jar*
bdlib-1.12.0.2-mc1.10.2.jar*
BetterBuildersWands-0.6.5-1.10r138+8595fc7.jar*
BiomesOPlenty-1.9.4-4.1.0.2052-universal.jar*
Botania-unofficial.r1.8-312.jar*
Chameleon-1.9.4-2.1.6.jar*
Charset-0.3.0-pre12.jar*
ChestTransporter-1.10-2.5.0.jar*
Chisel-MC1.9.4-0.0.6.33.jar*
chiselsandbits-11.5.jar*
CodeChickenCore-1.10.2-2.1.8.79-universal.jar*
combathelp-1.9.4-1.0.3.jar*
compactstorage-2.1-1.9.4.jar*
CompassCoords-1.10-3.1.6.jar*
ComputerCraft1.80pr0.jar*
CookingForBlockheads-mc1.9.4-3.1.26.jar*
CustomMainMenu-MC1.9.4-2.0.1.jar*
deepresonance-1.9.4-1.2.0beta35.jar*
ElecCore-1.9.4-1.5.292.jar*
EnderCore-1.10.2-0.4.1.40-beta.jar*
EnderIO-1.10.2-3.0.1.70_beta.jar*
EnderStorage-1.10.2-2.1.3.73-universal.jar*
endertanks-1.10.0-1.4.1.jar*
environmentaltech-1.10-0.10.1a.jar*
extrautils2-1.10-alpha-1.0.0.jar*
flatcoloredblocks-mc1.10-v4.1.jar*
forestry_1.10.2-5.1.12.198.jar*
ForgeAnalytics-1.10-0.0.0.19.jar*
FTBLib-1.9.4-2.1.0-pre2.jar*
FTBUtilities-1.9.4-2.1.0-pre2.jar*
gendustry-1.6.4.8-mc1.10.2.jar*
generators-0.9.20.4-mc1.10.2.jar*
GraveStone Mod 1.5.5.jar*
ImmersiveEngineering-0.10-38.jar*
InventoryTweaks-1.61-58.jar*
ironchest-1.10.2-7.0.6.794.jar*
itemscroller-1.10.2-0.5.2.jar*
jei_1.10.2-3.7.6.231.jar*
journeymap-1.10.2-5.2.4-unlimited.jar*
JustEnoughResources-1.10.2-0.4.6.41.jar*
Mantle-1.10-0.10.3.jar*
mcjtylib-1.10-1.9.8.jar*
Mekanism-1.10.2-9.1.0.288.jar*
MekanismGenerators-1.10.2-9.1.0.288.jar*
MekanismTools-1.10.2-9.1.0.288.jar*
MmmMmmMmmMmm-1.11.jar*
Pam's HarvestCraft 1.9.4c.jar*
PersistentBits-1.10.2-1.0.3.jar*
Quark-beta-44.jar*
RandomThings-MC1.10-3.7.4.jar*
refinedstorage-0.8.10.jar*
ResourceLoader-MC1.9.4-1.5.1.jar*
rftools-1.10-5.06.jar*
roots-0.107.jar*
shetiphiancore-1.10.0-3.3.0.jar*
SolarFluxReborn-1.9.4_0.7b.jar*
SoulShards-TOW-1.10.2-2.5.3-37.jar*
StorageDrawers-1.9.4-3.2.0.jar*
supercraftingframe-1.10.2-1.2.0.85.jar*
TConstruct-1.10-2.3.3a.jar*
theoneprobe-1.10-1.0.13.jar*
valkyrielib-1.10-0.10.1.jar*
Waila-1.7.0-B3_1.9.4.jar*
WailaHarvestability-mc1.10-1.1.7.jar*
Wawla-1.10-2.3.0.193.jar*
![transparent-gui-1](https://cloud.githubusercontent.com/assets/7588838/18194041/bf79712e-7097-11e6-9d5f-99d410c9dc3e.png)
![transparent-gui-2](https://cloud.githubusercontent.com/assets/7588838/18194042/bf94db62-7097-11e6-84a3-ab70b186bbd3.png)
![transparent-gui-3](https://cloud.githubusercontent.com/assets/7588838/18194043/bf97c296-7097-11e6-900a-7ee69b7ca75e.png)

**Comment by BluSunrize** [comment]
Can you reproduce this in a new world, with an empty inventory and JEI hidden?

**Comment by croftyraider** [comment]
Ok, more data - I was able to cause it to come and go via JEI - I had put in @stora into JEI for searching so it was showing recipes from various mods including EnderStorage which is also a new mod I had recently added... Changing my search terms to "large meal" (I was making food from harvest craft) cause the transparency issue to stop. So I switched the search term back to @stora and bam it was back again, plus I could cause it to come and go depending on where my mouse was...

So, suggestions on where to point the issue now? If it's one of chickenbones things, then bah I'll just remove the mod.

**Comment by BluSunrize** [comment]
You should try to narrow down which item on that page does it.
Either first off, I would recommend completing hte mod ID. I think you ahve refined storage and storage drawers there, right? My guess is that Refined Storage does it. After confirming which of the mods does it, you then try to narrow it down to which item specifically, so from all the items you see on that JEI page, you search SPECFICALLY for them. And see if the bug reappears.
After you have confirmed what item does it, report to the mod author 
(and do let me know which one it was, I am quite curious myself!)

**Comment by croftyraider** [comment]
Running out of time for today, but I'll get back to you tomorrow - it's something, but not you (none of your recipes were on display in JEI), but I'll let you know what it is when I track it down. One of the mods is doing something goofy with JEI or JEI has a bug. I'll see if JEI has been updated and test that out as well.

I did try removing enderstorage and that made it go away with my search term (note that refined storage was still in the list). But just searching for "pouch" did not demonstrate the bug when enderstorage was in the mod list, so I'm guessing now that JEI is the culprit... maybe... :)

Sorry to bother you!

**Comment by BluSunrize** [comment]
please do consider that Enderstorage also adds the chest and tank, not just the pouch. It could be any of them. I'm personally inclined to believe that this is not a JEI bug, mostly, because I know the guy who writes JEI, and his code is rock solid otherwise =P

**Comment by BluSunrize** [comment]
Issue closed, since it's most likely not IE related, but feel free to keep me up-to-date

**Comment by croftyraider** [comment]
Ok, it took a bit of doing, but I managed to test with the latest JEI (had to update forge and then all heck broke loose). The bug as it stands is still there, and I can't seem to point a finger at a single mod. Here is the reduced modlist using forge 2075:

Chameleon-1.9.4-2.1.6.jar*
CodeChickenCore-1.10.2-2.1.8.79-universal.jar*
EnderStorage-1.10.2-2.1.3.73-universal.jar*
jei_1.10.2-3.9.0.244.jar*
journeymap-1.10.2-5.2.4-unlimited.jar*
JustEnoughResources-1.10.2-0.4.6.41.jar*
refinedstorage-0.9.4.jar*
StorageDrawers-1.9.4-3.2.0.jar*
Waila-1.7.0-B3_1.9.4.jar*
WailaHarvestability-mc1.10-1.1.7.jar*
Wawla-1.10-2.3.0.193.jar*

What I do know is that it does happen repeatably, and that somehow by having items from EnderStorage, RefinedStorage and StorageDrawers showing on the JEI screen the bug appears (search term of @stor does it every time). Just having one mod's items does not show the issue. I did notice this time that when the tooltip is flown for the items that the transparency on the GUI disappears...

You know better than I how JEI renders the various items whether it's doing it itself or calling into the Mod classes to do it, but it kinda smells like a JEI issue to me. Your thoughts before I go bug mezz?
